caf/scripts/BasrevnuCom.py

```diff
@@ -11,23 +11,23 @@
 
 df = pd.read_csv('source/BasrevenuCom2009.csv', sep=";")
 
-df.columns = ['Communes', 'Codes_Insee', 'NB_Allocataires_2009', 
-              'ALL_bas_revenu_2009', 'Pers_bas_revenu_2009']
+df.columns = ['Communes', 'Codes_Insee', 'NB_Allocataires_2009',
+              'ALL_bas_revenu_2009', 'Pers_bas_revenu_2009', 'NB_pers_couv_ress']
 
 files = glob.glob('source/BasrevenuCom*')
 
 for path_file in files:
     year = str(path_file[-8:-4])
     if (year != '2009'):
         df_temp = pd.read_csv(path_file, sep=';')
-        
+
         # Rename Col with year
-        year_col = ['Communes', 'Codes_Insee']
+        year_col = ['Communes', 'Codes_Insee', 'NB_pers_couv_ress']
```

Comment:
"NB_pers_couv_ress" is a features and we have to rename it with _yyyy (year of files like line 14)